Source/Core/DolphinQt/BBAClient.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt/BBAClient.h"
+
+#include <QTcpSocket>
```

Comment:
These includes should be ordered alphabetically. Ditto for the DolphinQt ones below as well.